src/command-bus.ts

```diff
@@ -58,7 +58,9 @@ export class CommandBus<CommandBase extends ICommand = ICommand>
     const commandId = this.getCommandId(command);
     const handler = this.handlers.get(commandId);
     if (!handler) {
-      throw new CommandHandlerNotFoundException(commandId);
+      throw new CommandHandlerNotFoundException(
+        Object.getPrototypeOf(command).constructor.name,
+      );
     }
```

Comment:
Could you add a new `getCommandName` method instead of doing this all inline?
```ts
private getCommandName(command: Type<CommandBase>): string {
    const { constructor } = Object.getPrototypeOf(command);
    return constructor.name as string;
}
```

Reply:
yes of course!

Reply:
one question, when I run format there is a lot of files which are not well formated, did you wish I creacte a P/R with all the changes? (And maybe can be add some workflow to run it)

Reply:
sounds good!

Reply:
can we merge this? I will create a different P/R (When I've got a little bit of time)